docs/features/logforwarding/outputs/otlp-forwarding.adoc

```diff
@@ -93,35 +93,53 @@ https://github.com/open-telemetry/opentelemetry-proto/blob/v1.3.1/examples/logs.
 [%header,format=csv]
 |===
 ViaQ,OTEL
-@timestamp,	                timeUnixNano
-N/A,	                    observedTimeUnixNano
-level,	                    severityNumber
-message,                    body
+@timestamp,                       timeUnixNano
+N/A,                            observedTimeUnixNano
+level,                            severityNumber
+message/structured,          body
 --,--
-openshift.cluster_id,             cluster.id
-hostname,	                      node.name
-kubernetes.namespace_name,	      k8s.namespace.name
-kubernetes.pod_name,	          k8s.pod.name
-kubernetes.pod_id,	              k8s.pod.uid
-kubernetes.container_name,	      k8s.container.name
-kubernetes.container_id,	      k8s.container.id
-log_source,	                      openshift.log.source
-log_type,	                      openshift.log.type
-url,                              url.full
-response.code,                    http.response.status.code
-verb,                             http.request.method
-systemd.t.PID,                    syslog.procid
-systemd.u.SYSLOG.FACILITY,        syslog.facility
-systemd.u.SYSLOG.IDENTIFIER,      syslog.identifier
-systemd.t.SYSTEMD.CGROUP,         system.cgroup
-systemd.t.SYSTEMD.INVOCATION.ID,  system.invocation.id
-systemd.t.SYSTEMD.SLICE,          system.slice
-systemd.t.SYSTEMD.UNIT,           system.unit
-systemd.t.CMDLINE,                system.cmdline
+openshift.cluster_id,             k8s.cluster.uid
+hostname,                         k8s.node.name
+kubernetes.container_name,        k8s.container.name
+kubernetes.namespace_name,        k8s.namespace.name
+kubernetes.pod_name,              k8s.pod.name
+kubernetes.container_id,          k8s.container.id
+kubernetes.pod_id,                k8s.pod.uid
+kubernetes.namespace_labels.*,    k8s.namespace.label.*
+kubernetes.labels.*,              k8s.pod.label.*
+openshift.labels.*,               openshift.label.*
+log_type,                         openshift.log.type
+log_source,                       openshift.log.source
+hostname,                         url.domain
```

Comment:
hostname is listed twice.  Do these need to be separated by log_type?

Reply:
They are indeed separated by log_type.    k8s.node.name is set for auditd logs as a resource attribute (grouped and reduced).    url.domain is set for API logs and is a logrecord attribute, so not grouped/reduced by.

Reply:
At some point, I can create a mapping for the specific fields within each log_type, and match it up with the ViaQ structs we have currently.